app/src/main/java/org/xbmc/kore/provider/MediaProvider.java

```diff
@@ -737,11 +737,14 @@ private SelectionBuilder buildQuerySelection(Uri uri, int match) {
                               .mapToTable(MediaContract.Songs.SONGID, MediaDatabase.Tables.SONGS)
                               .mapToTable(MediaContract.Songs.TITLE, MediaDatabase.Tables.SONGS)
                               .mapToTable(MediaContract.Songs.ALBUMID, MediaDatabase.Tables.SONGS)
-                              .mapToTable(MediaContract.Songs.UPDATED, MediaDatabase.Tables.SONGS)
-                              .mapToTable(MediaContract.Songs.THUMBNAIL, MediaDatabase.Tables.SONGS)
```

Comment:
Any reason for leaving this out?
Not really relevant i guess, but was wondering why.